CliClient/tests/services_rest_Api.js

```diff
@@ -326,4 +326,28 @@ describe('services_rest_Api', function() {
 		expect(response3.length).toBe(2);
 	}));
 
+	it('should update tags when updating notes', asyncTest(async () => {
+		const tag1 = await Tag.save({ title: 'mon étiquette 1' });
+		const tag2 = await Tag.save({ title: 'mon étiquette 2' });
+		const tag3 = await Tag.save({ title: 'mon étiquette 3' });
+		const note = await Note.save({
+			title: 'ma note un',
+			tags: `${tag1.title},${tag2.title}`,
```

Comment:
Does this add the tags correclty?  Perhaps you could start the testing by getting the note tags to check the note is set up as you intend?

Reply:
Yes, it does add the tags in the note (checked in the desktop app). But I haven't checked the note in the unit test. I will rewrite the unit test to make sure it tests the notes rather than the response.

Reply:
Since you have the response checks, they are still useful.  So just add the new checks, so it checks both.